backend/src/database/employmentContract/employmentContract.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Controller, Get, Post, Param, Put, Body } from '@nestjs/common';
+import { EmploymentContractService } from './employmentContract.service';
+import {EmploymentContract} from './interfaces/employmentContract.interface';
+
+@Controller('db/employmentContract')
+export class EmploymentContractController {
+    constructor(private readonly employmentContractService: EmploymentContractService) {}
+
+    // TODO this is a skeleton
+    // TODO implementation needed
+    @Get(':user_id/:id')
+    async findOne(@Param('user_id') user_id, @Param('id') id) {
```

Comment:
Es wäre wichtig hier die Parameter zu prüfen. Also in diesem Fall, müssen user_id und id numerisch sein. Strings geben vermutlich keinen Sinn.

Grundsätzlich geht das, indem du ein .dto.ts file anlegst und dort mit Hilfe des class-validators die typen validierst. Du kannst dir als Beispiel den lang.controller.ts sowie lang.dto.ts anschauen. Ich bin mir aber nicht sicher, ob du damit einzelne variablen prüfen kannst.

Ich bin mir aber nicht sicher ob wir diese Endpoints benötigen (siehe Slack).